.travis.yml

```diff
@@ -1,49 +1,76 @@
-sudo: required
-
-language: generic
+sudo: 'required'
 
 services:
-  - docker
+  - 'docker'
+
+language:
+  - 'bash'
+
+env:
+  global:
+    - NODE_RED_VERSION=x.y.z
+    - TARGET=raymondmm/node-red
```

Comment:
just checking - this is just a temp dir for build ?

Reply:
NODE_RED_VERSION is a environment variable which gets filled by update_version.sh

TARGET is a environment variable which holds the target Docker repo, in this case raymondmm/node-red, but will change to nodered/node-red-docker or even better to nodered/node-red (without "-docker"), but this means a new repo on Docker hub. In my humble opinion `nodered/node-red-docker` is a pleonasm and should be left as is, while all new stuff should go into `nodered/node-red`.